core/blockchain.go

```diff
@@ -110,6 +110,7 @@ const (
 	blockCacheLimit     = 256
 	diffLayerCacheLimit = 1024
 	receiptsCacheLimit  = 10000
+	blobsCacheLimit     = 10000
```

Comment:
would it be too much? 10000*3*128KB -> 3.6GB.
As it is mainly used by DA check on block import, maybe 1000 could enough